_bioinfcomputing/software_languages.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Getting Started with Common Languages and Tools
+last_modified_at: 2019-04-24
+primary_reviewers: k8hertweck
+---
+
+## Training Resources 
+
+Fred Hutch offers training resources for each of the common languages and tools listed below. For course descriptions and session notifications, go to Hutch Learning [here](http://www.fredhutch.io/). New class offerings will also be posted in the Coop Newsletter. Email `thecoop` to join the mailing list.
```

Comment:
I'd revise this section as follows:

Fred Hutch offers training resources for many of the common languages and tools listed below. Course descriptions can be found at [fredhutch.io](http://www.fredhutch.io/), and course registration is tracked in Hutch Learning (only accessible to Fred Hutch employees through CenterNet; search for "fredhutch.io" to see all relevant courses). New class offerings will also be posted in the Coop Newsletter. Email `thecoop@fredhutch.org` to join the mailing list.

Reply:
Updated except for the email address, because the wiki convention is to hide the details.